libraries/chain/fork_database.cpp

```diff
@@ -251,20 +226,14 @@ namespace eosio::chain {
          fc::raw::pack( out, head->id() );
       } else {
          elog( "head not set in fork database; '${filename}' will be corrupted",
-               ("filename", fork_db_dat) );
+               ("filename", fork_db_file) );
       }
 
       index.clear();
    }
 
    template<class bsp>
-   fork_database<bsp>::~fork_database() {
-      my->close_impl();
-   }
```

Comment:
Why do we not need the destructor anymore?

Reply:
Because it is handled by `fork_database` now instead of `fork_database_t`.